### PR TITLE
Temporary fix for Brave Ads frequency cap bug

### PR DESCRIPTION
**Pull request by tmancey**
See https://github.com/brave/brave-variations/pull/517 for STR

### CODE REVIEW

**Comment by btlechowski** [comment]
Good to uplift

Verification passed on

Brave | 1.48.157 Chromium: 110.0.5481.77 (Official Build) (64-bit) 
-- | --
Revision | 65ed616c6e8ee3fe0ad64fe83796c020644d42af-refs/branch-heads/5481@{#839}
OS | Ubuntu 18.04 LTS



Verified griffin study is in the seed:
image


Verified griffin study is applied:
```
[29224:29224:0207/113127.683768:VERBOSE2:permission_rule_util.cc(25)] User was inactive
[29224:29224:0207/113127.691225:VERBOSE1:new_tab_page_ad_serving.cc(65)] New tab page ad not served: Not allowed due to permission rules
[29224:29224:0207/113127.702701:VERBOSE1:ads_service_impl.cc(902)] Failed to prefetch new tab page ad

```

Verified ntpsi is served when threshold is reached:
```
[29224:29224:0207/113633.497343:VERBOSE1:eligible_new_tab_page_ads_v2.cc(34)] Get eligible new tab page ads
[29224:29224:0207/113633.522012:VERBOSE1:eligible_new_tab_page_ads_v2.cc(120)] 1 eligible ads out of 1 ads
[29224:29224:0207/113633.527753:VERBOSE1:new_tab_page_ad.cc(64)] Opportunity arose to serve a new tab page ad
[29224:29224:0207/113633.528142:VERBOSE1:new_tab_page_ad_serving.cc(94)] Found 1 eligible ads
[29224:29224:0207/113633.528730:VERBOSE1:new_tab_page_ad_serving.cc(132)] Served new tab page ad:
  placementId: 2088b9cc-1310-450d-beaa-c802fb933d6e
  creativeInstanceId: 700b6be4-2914-4462-a681-f1d4bf8a4fc2
  creativeSetId: 4f47e9d6-6828-485a-8984-707e456f8836
  campaignId: 04a93367-6c27-497a-a66d-5ea2eb7a5110
  advertiserId: 37807725-67d9-42fb-b8b7-d68cf56b254f
  segment: health & fitness
  companyName: bbondy
  imageUrl: https://mobile-data-dev.s3.brave.software/assets/ntp-si/e34d8f59-7e96-48b8-828e-3a523ea4f7ee.png
  alt: bbondy logo
  targetUrl: https://brianbondy.com/
  wallpaper:
    imageUrl: https://mobile-data-dev.s3.brave.software/assets/ntp-si/f8280622-448a-4077-93f3-05a8af91aa14.jpg
    focalPoint:
      x: 0
      y: 720
[29224:29224:0207/113633.574233:VERBOSE3:new_tab_page_ad_event_served.cc(19)] Served new tab page ad with placement id 2088b9cc-1310-450d-beaa-c802fb933d6e and creative instance id 700b6be4-2914-4462-a681-f1d4bf8a4fc2
[29224:29224:0207/113633.622787:VERBOSE6:new_tab_page_ad_event_served.cc(28)] Successfully logged new tab page ad served event

```

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/518#issuecomment-1420632227.

**Pull request merged**
